### PR TITLE
fix documentation typos

### PR DESCRIPTION
**Pull request by IlyaFaer**

### CODE REVIEW

**Comment by IlyaFaer** [comment]
@rudolfix

**Review by burnash — APPROVED** [review]
Thanks for spotting this, @IlyaFaer

**Review by rudolfix — CHANGES_REQUESTED** [review]
thx for the fixes and for the review!
the tests for docs examples are not passing. the reason is that @IlyaFaer submitted PR from his fork of dlt and in that case, github actions do not have access to the secrets

I think I already asked to mark mongo tests to not run on fork

@IlyaFaer please do the above :) you can find a @skipifgithubfork decorator and mark the failing tests

in case they are already marked we need to fix the decorator. I know there were some problems with it

thanks :)

File: .github/workflows/test_doc_snippets.yml [hunk 1]
```diff
@@ -24,6 +24,8 @@ env:
   RUNTIME__SLACK_INCOMING_HOOK: ${{ secrets.RUNTIME__SLACK_INCOMING_HOOK }}
   # Mongodb url for nested data example
   MONGODB_PIPELINE__SOURCES__CONNECTION_URL: ${{ secrets.MONGODB_PIPELINE__SOURCES__CONNECTION_URL }}
+
+  IS_FORK: ${{ github.event.pull_request.head.repo.fork }}
```
Comment (IlyaFaer) [inline .github/workflows/test_doc_snippets.yml:28]: This var is intended for forks detection, so it's functionality is guaranteed

File: tests/utils.py [hunk 2]
```diff
@@ -247,9 +247,8 @@ def skip_if_not_active(destination: str) -> None:
 
 def is_running_in_github_fork() -> bool:
     is_github_actions = os.environ.get("GITHUB_ACTIONS") == "true"
-    head_ref = os.environ.get("GITHUB_HEAD_REF", "")
-    repo = os.environ.get("GITHUB_REPOSITORY", "")
-    return is_github_actions and ":" in head_ref and not head_ref.startswith(repo.split("/")[0])
```
Comment (IlyaFaer) [inline tests/utils.py:252]: These variables are not very reliable (like any paths on GitHub):

image

**Review by rudolfix — CHANGES_REQUESTED** [review]
I like it!
Please add comments both to the decorator and to the workflow yaml that explain how they work together - that you need to explicitly define the variable in order for it to work

then I'm good with it and we can merge

**Comment by IlyaFaer** [comment]
@rudolfix, okay, pushed

**Review by rudolfix — APPROVED** [review]
LGTM!

**Pull request merged**